shared/src/persistence/dynamo/users/createUser.js

```diff
@@ -130,6 +135,18 @@ exports.createUser = async ({ applicationContext, password, user }) => {
     userId = response.Username;
   }
 
+  if (disableCognitoUser) {
+    await applicationContext.getCognito().adminDisableUser({
+      UserPoolId: userPoolId,
+      Username: userId,
+    });
+  }
+
+  // note: this will only be called on local envs
```

Comment:
Couldn't you flag this on `process.env = local` or something like that to guarantee this is only called locally?

Reply:
We've removed this block entirely!

Reply:
Nice!